implants/lib/eldritch/src/pivot.rs

```diff
@@ -78,9 +78,18 @@ fn methods(builder: &mut MethodsBuilder) {
         if false { println!("Ignore unused this var. _this isn't allowed by starlark. {:?}", this); }
         port_scan_impl::port_scan(starlark_heap, target_cidrs, ports, protocol, timeout)
     }
-    fn arp_scan(this:  PivotLibrary, target_cidrs: Vec<String>) ->  anyhow::Result<Vec<String>> {
-        if false { println!("Ignore unused this var. _this isn't allowed by starlark. {:?}", this); }
-        arp_scan_impl::arp_scan(target_cidrs)
+    fn arp_scan<'v>(
+        this: PivotLibrary,
+        starlark_heap: &'v Heap,
+        target_cidrs: Vec<String>,
+    ) -> anyhow::Result<Vec<Dict<'v>>> {
+        if false {
+            println!(
+                "Ignore unused this var. _this isn't allowed by starlark. {:?}",
+                this
+            );
+        }
+        arp_scan_impl::arp_scan(starlark_heap, target_cidrs)
```

Comment:
May wait on this want to figure out  the cargo clippy PR before merging any new formatted code.